echo_server.py

```diff
@@ -21,10 +26,74 @@ def echo_server():
                 if len(response) < 32:
                     done = True
                 text_received = "{}{}".format(text_received, response)
-            conn.sendall(text_received)
+            if text_received:
+                response_user = parase_request(text_received)
+            conn.sendall(response_user)
     except KeyboardInterrupt:
        server_socket.close()
 
+def response_ok(content, body_text):
+    first_line = 'HTTP/1.1 200 OK'
+    timestamp = 'Date: ' + email.utils.formatdate(usegmt=True)
+    header = 'Content-Type: {}'.format(content)
+    body = body_text
+    size = str(sys.getsizeof(body))
+    end = '\r\n'
+    response = [first_line, timestamp, header, size, body, end]
+    return end.join(response)
+
+def response_error(error_code, error_message):
+    first_line = 'HTTP/1.1 {} {}'.format(error_code, error_message)
+    timestamp = 'Date: ' + email.utils.formatdate(usegmt=True)
+    header = 'Content-Type: text/plain'
+    body = '{} {}'.format(error_code, error_message)
+    size = str(sys.getsizeof(body))
```

Comment:
As it is, this is not a proper HTTP response header, it's just a string that holds a number.  Make it into a Content-Length header.